kettle-plugins/sqoop/src/main/java/org/pentaho/big/data/kettle/plugins/sqoop/ui/SwtLabelOrLink.java

```diff
@@ -20,7 +20,7 @@
  *
```

Comment:
Is this class used any more?  I think we pick up the SwtLabelOrLink from the xul library now, iirc.